.github/workflows/bsi-update.yml

```
@@ -0,0 +1,69 @@
+# This workflow runs "npm i" to see if any dependencies have updated.  If so, it opens a PR and approves it.
+
+name: BSI Update
+
+on:
+  workflow_dispatch: # manual trigger for testing
+
+jobs:
+  build:
+    # self-hosted doesn't work for public repos (security), so need to use ubuntu
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```

```suggestion
        uses: Brightspace/third-party-actions@actions/checkout
```

This isn't enforced yet but will be "soon". [Check the docs for the rationale](https://github.com/Brightspace/third-party-actions#why).

---

Makes total sense